src/components/PeoplePages/NewPerson/NewPerson.tsx

```diff
@@ -0,0 +1,251 @@
+  const [born, setBorn] = useState('');
+  const [died, setDied] = useState('');
+  const [fatherName, setFatherName] = useState('');
+  const [motherName, setMotherName] = useState('');
+
+  const [nameError, hasNameError] = useState(false);
+  const [bornError, hasBornError] = useState(false);
+  const [diedError, hasDiedError] = useState(false);
+  const [ageError, hasAgeError] = useState(false);
+
+  const handleSubmit = (event: React.FormEvent) => {
+    event.preventDefault();
+
+    const mother = people
+      .find(candidate => candidate.name === motherName);
+
+    const father = people
+      .find(candidate => candidate.name === fatherName);
+
+    onSetPeople([...people, {
+      name,
+      sex,
+      born: +born,
+      died: +died,
+      fatherName,
+      motherName,
+      slug: `${name.split(' ').join('-')}-${born}`,
+      father: father || null,
+      mother: mother || null,
+    }]);
+
+    setName('');
+    setSex('m');
+    setBorn('');
+    setDied('');
+    setFatherName('');
+    setMotherName('');
+  };
+
+  const handleErrors = () => {
+    if (!name) {
+      hasNameError(true);
+    }
+
+    if (+born < 1400 || +born > 2022) {
+      hasBornError(true);
+    }
+
+    if (+died < 1400 || +died > 2022) {
+      hasDiedError(true);
+    }
+
+    if (+died - +born > 150 || died < born) {
+      hasAgeError(true);
+    }
+  };
+
+  return (
+    <form
+      action="GET"
+      className="form is-flex is-justify-content-center"
+      onSubmit={handleSubmit}
+    >
+      <nav
+        className="panel is-flex is-flex-direction-column"
+        style={{ width: '700px' }}
+      >
+        <p className="panel-heading">
+          New person data
+        </p>
+        <div
+          className="is-flex is-flex-direction-column"
+          style={{ width: '600px' }}
+        >
+          <div className="panel-block is-flex is-justify-content-space-between">
+            <div>
+              <span>
+                Name:
+                <input
+                  type="text"
+                  value={name}
+                  className={classNames('input', {
+                    'is-danger': nameError,
+                  })}
+                  placeholder="name"
+                  onBlur={handleErrors}
+                  onChange={({ target }) => {
+                    setName(target.value);
+                    hasNameError(false);
+                  }}
+                />
+              </span>
+              {nameError && (
+                <span>Enter the name</span>
+              )}
+            </div>
+
+            <div>
+              Sex:
+              <span className="panel-block">
+                <label className="table__label is-capitalized">
+                  <input
+                    type="radio"
+                    name="sex"
+                    className="radio"
+                    value="f"
+                    onChange={() => setSex('f')}
+                  />
```

Comment:
I don't really need it. This is a radiobutton, I put `checked` on another button just to make one of them checked from the start

Reply:
Yes, that makes sense. But if you need the user not to forget to make a choice, it will be better not to make any of them selected at the start, and make them required